SystemInformer/tokprp.c

```diff
@@ -527,14 +528,21 @@ PPH_STRING PhGetElevationTypeString(
 
     PhInitializeStringBuilder(&sb, 13);
 
-    PhAppendStringBuilder2(&sb, IsElevated ? L"Yes" : L"No");
+    if (IsTokenAccessible)
+    {
+        PhAppendStringBuilder2(&sb, IsElevated ? L"Yes" : L"No");
 
-    if (ElevationType == TokenElevationTypeFull)
-        PhAppendStringBuilder2(&sb, L" (Full)");
-    else if (ElevationType == TokenElevationTypeLimited)
-        PhAppendStringBuilder2(&sb, L" (Limited)");
+        if (ElevationType == TokenElevationTypeFull)
+            PhAppendStringBuilder2(&sb, L" (Full)");
+        else if (ElevationType == TokenElevationTypeLimited)
+            PhAppendStringBuilder2(&sb, L" (Limited)");
+        else
```

Comment:
This assumes the `0` and `TokenElevationTypeDefault` are equivalent, which is not the case.

Reply:
Modified